From 6b714f558b6cfb9f25c58b70eeb134f3bfa1bebd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lum=C3=ADr=20=27Frenzy=27=20Balhar?=
 <frenzy.madness@gmail.com>
Date: Wed, 30 Nov 2022 22:47:56 +0100
Subject: [PATCH] Drop dependency on py

pytest 7.2 and tox 4 no longer depend on py and it's not needed here as well.
---
 testing/test_iniconfig.py | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/testing/test_iniconfig.py b/testing/test_iniconfig.py
index fe12421..027949d 100644
--- a/testing/test_iniconfig.py
+++ b/testing/test_iniconfig.py
@@ -1,4 +1,3 @@
-import py
 import pytest
 from iniconfig import IniConfig, ParseError, __all__ as ALL
 from iniconfig import iscommentline
@@ -100,7 +99,7 @@ def parse(input):
 
 
 def parse_a_error(input):
-    return py.test.raises(ParseError, parse, input)
+    return pytest.raises(ParseError, parse, input)
 
 
 def test_tokenize(input, expected):
@@ -135,7 +134,7 @@ def test_section_cant_be_empty():
     assert excinfo.value.lineno == 0
 
 
-@py.test.mark.parametrize('line', [
+@pytest.mark.parametrize('line', [
     '!!',
     ])
 def test_error_on_weird_lines(line):
-- 
2.39.0

